lib/install/management/rbac.go

```diff
@@ -281,7 +176,7 @@ func (am *AuthzManager) setupPermissions(ctx context.Context) ([]RBACResourcePer
 	}
 
 	// Find bridged networks
-	bridgeNet, ok := am.configSpec.Networks["bridge"]
+	bridgeNet, ok := am.configSpec.Network.ContainerNetworks["bridge"]
```

Comment:
What's the purpose of this change?

Reply:
This is a bug, fix. Taking the bridgeNet from the Networks map works with the Sim but bot with real VC.